README.md

```diff
@@ -122,7 +122,7 @@ mkdir split_keys
 # E.g., keystore-0.json keystore-0.txt
 
 # Split these keystores into "n" (--nodes) key shares with "t" (--threshold) as threshold for a distributed validator
-docker run --rm -v $(pwd):/opt/charon obolnetwork/charon:v0.14.0 create cluster --split-existing-keys --split-keys-dir=/opt/charon/split_keys --threshold 4 --nodes 6
+docker run --rm -v $(pwd):/opt/charon obolnetwork/charon:v0.14.0 create cluster --name="mycluster" --withdrawal-addresses="0x000000000000000000000000000000000000dead" --fee-recipient-addresses="0x000000000000000000000000000000000000dead" --split-existing-keys --split-keys-dir=/opt/charon/split_keys --threshold 4 --nodes 6
```

Comment:
Hope this command is directly taken from the latest docs - https://docs.obol.tech/docs/int/quickstart/quickstart-alone#step-2-create-the-artifacts-needed-to-run-a-dv-cluster ?

Reply:
Don't think we should use 0xdead addresses, rather specify that the user should provide a proper address.

```
WITHDRAWAL_ADDR=<put your withdrawal here>
FEE_RECIPIENT_ADDR=<...>
docker run --rm -v $(pwd):/opt/charon obolnetwork/charon:v0.14.0 create cluster --name="mycluster" --withdrawal-addresses="${WITHDRAWAL_ADDR}" --fee-recipient-addresses="${FEE_RECIPIENT_ADDR}" --split-existing-keys --split-keys-dir=/opt/charon/split_keys --threshold 4 --nodes 6
```